project_gtd/tests/test_project_gtd.py

```diff
@@ -42,16 +45,33 @@ def test_timebox_weekly(self):
         self.plan.process()
         self.assertTrue(self.task.timebox_id.id == self.timebox_weekly.id)
 
-    def test_timebox_empty(self):
+    def test_timebox_empty_close(self):
         """
         Check if timebox is empty after emptiying task.
         """
+        self.plan.process()
+        context = {
+            "active_model": "project.gtd.timebox",
+            "active_ids": [
+                self.timebox_weekly.id,
+            ],
+            "active_id": self.timebox_weekly.id,
+        }
+        self.project_timebox_empty.with_context(context).process()
+        self.assertFalse(self.task.timebox_id)
+
+    def test_timebox_empty_up(self):
+        """
+        Check if timebox is empty after emptiying task.
```

Comment:
It is intended this docstring is equal to the previous one?
Also, could you please avoid code repetition?

Reply:
I'll check, thanks

Reply:
This docstring seems not right yet. Could you please check it?

Reply:
done